packages/augur-ui/src/modules/market-cards/common.tsx

```diff
@@ -260,7 +247,7 @@ export const OutcomeGroup = (props: OutcomeGroupProps) => {
             min={props.min}
             max={props.max}
             lastPrice={
-              outcomesShow[0].price ? outcomesShow[0].lastPricePercent : null
+              outcomesShow[YES_NO_YES_ID].price ? formatNumber(outcomesShow[YES_NO_YES_ID].price) : null
```

Comment:
better to use scalar constants. 

export const SCALAR_DOWN_ID = 1;
export const SCALAR_UP_ID = 2;